Dockerfile

```diff
@@ -1,3 +1,18 @@
-FROM alpine:latest
+FROM dwpdigital/python-boto-behave
```

Comment:
Why not `FROM python:3.6-alpine` ?

Reply:
You are correct, needed boto and never thought about it past that, boto is now installed via pip in dockerfile and source is changed as you said.